third_party/istio-1.0.6/istio-lean.yaml

```diff
@@ -82,8 +82,7 @@ data:
             - destinationrules
             - envoyfilters
             - gateways
-            # disabled per @costinm's request
```

Comment:
Is there an issue that was fixed here or should this still be disabled?

Reply:
This file is automatically generated so the comment was probably removed upstream.